### PR TITLE
Issue 10363: Constants and config processing

### LINKED ISSUE #10363: acmeCA-2.0: Process acme config, fixes to find feature
Add constants and code to process the acmeCA config. Update so feature and bundle is found when configured.

For #9017

### PR DESCRIPTION
**Pull request by kristip17**
Fixes #10363

### CODE REVIEW

**Comment by kristip17** [comment]
@build

**Comment by kristip17** [comment]
@run-libby-bot

**Comment by kristip17** [comment]
#build

**Comment by kristip17** [comment]
#run-libby-bot

File: dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/config/AcmeConfig.java [hunk 1]
```diff
@@ -10,10 +10,40 @@
  *******************************************************************************/
 package com.ibm.ws.security.acme.config;
 
+import java.util.Set;
```
Comment (tevans78) [inline dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/config/AcmeConfig.java]: Copyright dates should be `2019, 2020`

Comment (kristip17) [reply dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/config/AcmeConfig.java]: Thanks, updated copyright.

**Comment by kristip17** [comment]
Green build and I push a commit with a copyright update.

**Comment by kristip17** [comment]
#run-libby-bot

File: dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/internal/AcmeProviderServiceImpl.java [hunk 2]
```diff
@@ -38,73 +37,86 @@
...
     }
 
     @Modified
     public void modify(Map<String, Object> properties) {
-        Tr.info(tc, " ******* JTM ******* AcmeProviderServiceImpl: inside modified () method");
+		initialize(properties);
     }
 
+	public void initialize(Map<String, Object> configProps) {
+		Tr.info(tc, "Initialize ACME");
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/internal/AcmeProviderServiceImpl.java]: Probably shouldn't be an 'info'.

File: dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/internal/AcmeProviderServiceImpl.java [hunk 3]
```diff
@@ -38,73 +37,86 @@
...
+		acmeClient.setChallengeRetries(challengeRetries);
+		acmeClient.setChallengeRetryWait(challengeRetryWait);
+		acmeClient.setOrderRetries(orderRetries);
+		acmeClient.setOrderRetryWait(orderRetryWait);
+
+	}
+
     @Deactivate
     public void deactivate(ComponentContext context, int reason) {
         Tr.info(tc, " ******* JTM ******* AcmeProviderServiceImpl: inside deactivate() method");
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/internal/AcmeProviderServiceImpl.java]: Remove this trace too.

File: dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/internal/AcmeProviderServiceImpl.java (hunk 2 above)
Comment (kristip17) [reply dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/internal/AcmeProviderServiceImpl.java]: Removed or commented out all the info's in this class.

File: dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/internal/AcmeProviderServiceImpl.java (hunk 3 above)
Comment (kristip17) [reply dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/internal/AcmeProviderServiceImpl.java]: Done

**Comment by kristip17** [comment]
#build

**Comment by kristip17** [comment]
#run-libby-bot

**Pull request merged**